server/src/main/java/org/candlepin/async/JobManager.java

```diff
@@ -912,8 +912,16 @@ private void synchronizeJobSchedule() throws SchedulerException {
 
         // Unschedule dead/invalid jobs
         for (JobKey key : unschedule) {
-            this.scheduler.deleteJob(key);
-            log.info("Removed existing schedule for job: {}", key.getName());
+            List<Trigger> jobTrigger = (List<Trigger>) this.scheduler.getTriggersOfJob(key);
```

Comment:
jobTrigger should be plural, since it's a collection.

Reply:
Fixed.